CHANGELOG.rst

```diff
@@ -5,6 +5,20 @@ community.aws Release Notes
 .. contents:: Topics
 
 
+v6.2.0
+======
+
+Release Summary
+---------------
+
+This release includes some new features for the ``community.aws.ec2_vpc_vpn`` and ``community.aws.api_gateway`` modules.
+
+Minor Changes
+-------------
+
+- api_gateway - add support for parameters `name`, `lookup`, `tags` and `purge_tags` (https://github.com/ansible-collections/community.aws/pull/1845).
```

Comment:
```suggestion
- api_gateway - add support for parameters ``name``, ``lookup``, ``tags`` and ``purge_tags`` (https://github.com/ansible-collections/community.aws/pull/1845).
```

Single ticks are interpreted as links.  Unfortunately this is "allowed" by the linters because it might be valid once the whole thing is compiled into a single document/site.  However, when we run the final site build this will trigger warnings about broken links